codegen/java-gen/src/main/scala/com/lightbend/akkasls/codegen/java/ValueEntitySourceGenerator.scala

```diff
@@ -23,6 +23,7 @@ import org.bitbucket.inkytonik.kiama.output.PrettyPrinterTypes.Document
 
 object ValueEntitySourceGenerator {
   import SourceGenerator._
+  import EntityServiceSourceGenerator.generateImports
```

Comment:
it's used by both ValueEntitySourceGenerator and EntityServiceSourceGenerator, should probably be in some third utility class but EntityServiceSourceGenerator anyway looked like it wasn't only about EventSourced

Reply:
Correct, the code for value entity and event sourced entity was previously in EntityServiceSourceGenerator and we started to split it into two different generators. 

At some point, we spined-off `ValueEntitySourceGenerator` and `EntityServiceSourceGenerator` should have been renamed to `EntitySourcedServiceSourceGenerator`, but we had other PRs in the making and it was better to postpone the rename to avoid conflicts.